pkg/ocm/ocm.go

```diff
@@ -28,37 +28,71 @@ type ServiceLog struct {
 	Username      string
 }
 
-type ocmClient struct {
+type Client struct {
 	// conn sdk.Connection undecided if necessary here or not
 	// Ocm Cluster Client
 	// clusterClient *cm v1.ClustersClient
-	clusterLogsClient *clv1.ClustersClusterLogsClient
+	clusterLogsClient    *clv1.ClustersClusterLogsClient
+	clusterLogsAddClient *clv1.ClusterLogsClient
 }
 
-// Not sure if I want this to be part of the ocmClient Struct yet.
+// NewClient Not sure if I want this to be part of the ocmClient Struct yet.
 // Any ways it needs to be exposed to the user for them to close the connection
-func NewClient(conn *sdk.Connection) ocmClient {
-	return ocmClient{conn.ServiceLogs().V1().Clusters().ClusterLogs()}
+func NewClient(conn *sdk.Connection) Client {
+	return Client{
+		clusterLogsClient:    conn.ServiceLogs().V1().Clusters().ClusterLogs(),
+		clusterLogsAddClient: conn.ServiceLogs().V1().ClusterLogs(),
+	}
+}
+
+func NewConnectionWithTemporaryToken(url, token string) (*sdk.Connection, error) {
```

Comment:
@rafael-azevedo I've added this new function to make connections with the `ocm token` output and a specified OCM URL.